sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -497,13 +551,15 @@ static void _netif_list(kernel_pid_t iface)
     line_thresh = _netif_list_flag(iface, NETOPT_CSMA, "CSMA  ",
                                    line_thresh);
     line_thresh += _LINE_THRESHOLD + 1; /* enforce linebreak after this option */
-    line_thresh = _netif_list_flag(iface, NETOPT_AUTOCCA, "AUTOCCA",
+    line_thresh = _netif_list_flag(iface, NETOPT_AUTOCCA, "AUTOCCA ",
```

Comment:
I added this space because otherwise all lines are displayed like:
```
AUTOCCAIQ_INVERTRX_SINGLE
```
instead of
```
AUTOCCA IQ_INVERT RX_SINGLE
```

Maybe I can move this change to its own commit